### ISSUE TITLE
Task: Add Task

### ISSUE THREAD

**Issue by mfuerni** [jguertl/SWWM#7]
Adds a new task to a category.

The user taps on the big add button (e. g. https://i.stack.imgur.com/s9RmX.png) in the category list or in the task list.
A new activity opens with input fields for:
- Title
- Description
- Due Date
- Reminder Date
- Category
- State

After entering the details of the task, the user presses OK/Save/... to save the task and return to the task list of the category where the task was inserted into.

Requires: #4, #6
